mailpoet/lib/Config/Migrator.php

```diff
@@ -1003,4 +1006,48 @@ private function setDefaultValueForLoadingThirdPartyLibrariesForExistingInstalls
 
     return true;
   }
+
+  private function migrateSegmentDisplaySettingsOnManageSubscriptionPage(): bool {
+    // skip the migration if the DB version is higher than 3.91.1 or is not set (a new installation)
```

Comment:
The comment here seems to be outdated. We are also planning an emergency release, so I guess we will need to increase also the tested version here from `3.101.1` to `3.102.1`.

Reply:
Noted